src/Okta.Sdk.Abstractions/IonApiError.cs

```diff
@@ -38,5 +51,14 @@ private string GetErrorSummary()
 
             return sbErrorSumary.ToString();
         }
+
+        private string GetFormValidationErrorSummary()
+        {
+            var jToken = JToken.Parse(GetRaw());
+            var errorMessages = jToken
+                                    .SelectTokens("$.remediation.value[*].value[*].form.value[*].messages.value[*].message")
```

Comment:
What's the reason we're using this form of traversal (this isn't the first example I've seen of this)?  I find this hard to read and understand, but maybe there's a reason for doing it this way versus, modeling the remediation, for example.

Reply:
Yes, that's how it's currently done in other SDKs. To my understanding we are doing it this way because the response format is expected to change. So we will have less refactoring to do in future.

Reply:
While this is a simple one liner that works, it is quite hard to follow.

Reply:
So keeping in mind we didn't want to model this particular response, what do you suggest @arvindkrishnakumar-okta? I'm curious to know if you guys have any other ideas.

Reply:
Didn't get to this ticket yet on the Java SDK side. We need to explore if Java has a similar framework that can grep for such form level errors inside a Json with a regex. If not, we'd have to tediously loop through each and every form and look for the values in it to see if there are any errors and then collect them.

cc: @sergiishamrai-okta

Reply:
Right, but if you had both approaches available in java, what would you use and why? @arvindkrishnakumar-okta

Reply:
I see no issues with this method right now. In the future, we will hopefully have an top level messages object we can use, but for now, this works

Reply:
I am fine with this approach that works at the moment (assuming we'll have root level messages soon).